webapp/portlet/src/main/webapp/vue-apps/common/main.js

```diff
@@ -19,6 +19,7 @@ import * as brandingService from './js/brandingService.js';
 import * as navigationService from '../common/js/NavigationService.js';
 import * as profileSettingsService from '../common/js/ProfileSettingsService.js';
 import * as profileLabelService from '../common/js/ProfileLabelService.js';
+import * as siteService from './js/SiteService.js';
```

Comment:
May be not used a lot as a common service. Can you move it in Hamburger menu please.

Reply:
fixed